odxtools/compumethods/readcompumethod.py

```diff
@@ -46,7 +46,8 @@ def _parse_compu_scale_to_linear_compu_method(scale_element,
     nums = coeffs.iterfind("COMPU-NUMERATOR/V")
 
     offset = computation_python_type(next(nums).text)
-    factor = computation_python_type(next(nums).text)
+    factor_el = next(nums, None)
+    factor = computation_python_type(factor_el.text if factor_el else '0')
```

Comment:
letting the scaling factor default to 0 does not really make sense because the physical value would not depend of the internal value anymore. did you mean 1?

Reply:
I though so at first, but the formula from ODX specs suggests otherwise, basically it could be that there are more than one denominator, and denominator[1] is the one that depends on internal value.

![image](https://user-images.githubusercontent.com/1814900/185181172-bb138cda-bf97-4766-94e6-befc18415080.png)

I think that the correct implementation should be done with a list of numerator and a list of denominator, but it would change the current API too much.

Reply:
> I though so at first, but the formula from ODX specs suggests otherwise, [...]

I see: all compu methods are basically seen as special cases of (piecewise?) rational functions. If we ever implement this fully, providing the inverse functions is going be a lot of fun...

> I think that the correct (generic) implementation should be done with a list of numerator and a list of denominator, but it would change the current API too much.

agreed. Maybe we should put it on a list for odxtools 2.0 features...